src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/PreReleaseUserViewModels.cs

```diff
@@ -23,7 +23,7 @@ public class PreReleaseUserInvitePlan
         public List<string> Invitable { get; } = new List<string>();
     }
 
-    public class PreReleaseUserInviteRequest
+    public class PreReleaseUserInviteRequestViewModel
```

Comment:
Same as above - would call this either `PreReleaseUserInviteViewModel` or `PreReleaseUserInviteRequest`.

Reply:
After Slack discussing, renaming to follow `BlahViewModel` convention